js/index.js

```diff
@@ -14,34 +14,52 @@ for (const item of skills){
     skillsList.appendChild(skill);
 }
 //Handle Message Form Submit
-const messageForm = document.getElementById('leave_message');
-messageForm.addEventListener("submit", function(event){    
+const messageForm = document.querySelector('form[name="leave_message"]');
+const input = messageForm.querySelector("input");
+messageForm.addEventListener("submit", (event) => {    
 
     event.preventDefault();
     
     const usersName = event.target.usersName.value;
     const usersEmail = event.target.usersEmail.value;
     const usersMessage = event.target.usersMessage.value;
     
+    const ul = document.getElementById('responds');
+    const li = document.createElement('li');
+    ul.appendChild(li);
+
     console.log('Name', usersName);
     console.log('Email', usersEmail);
     console.log('Message', usersMessage);
 
-    messageForm.reset();
- }, true);
+    const messageSection = document.getElementById('messages');
```

Comment:
Much easier to create the inner html of the list item using an interpolated string.

``
        <a href="mailto:${userEmail}">${userName}</a>
        <span>${userMessage}</span>
    ``